plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -751,14 +751,18 @@ private boolean isCollatable(JdbcColumnHandle column)
         if (column.getColumnType() instanceof CharType || column.getColumnType() instanceof VarcharType) {
             String jdbcTypeName = column.getJdbcTypeHandle().getJdbcTypeName()
                     .orElseThrow(() -> new TrinoException(JDBC_ERROR, "Type name is missing: " + column.getJdbcTypeHandle()));
-            // Only char (internally named bpchar)/varchar/text are the built-in collatable types
```

Comment:
Move this comment to the new method you added. `bpchar` isn't obvious and why these specific types are there isn't obvious otherwise.

Reply:
Moved.